src/data/python-quiz.ts

```diff
@@ -1,4 +1,15 @@
 const pythonQuiz = [
+  {
+    Question: 
+      "One line expression function in python is called as?",
+    Answer: "Lambda function",
+    Distractor1: "Lambda function",
```

Comment:
distractor are only for wrong answers

```suggestion
    Distractor1: "type function",
```